src/harness/testcases/testdata/sas_impl_record_0.json

```diff
@@ -0,0 +1,23 @@
+{
+  "id": "sas/0/0",
```

Comment:
Maybe we want to make this a little more realistic?

Also, the prefix has changed to "sas_impl/" (see the latest version of the SAS-SAS doc that Greg just sent out).